anitya/lib/exceptions.py

```diff
@@ -115,7 +115,6 @@ class RateLimitException(AnityaException):
     def __init__(self, reset_time):
         self._reset_time = reset_time
 
-    @property
     def reset_time(self):
```

Comment:
Ah, what I meant was just delete this method and change the name from ``self._reset_time`` to ``self.reset_time``

Reply:
I wouldn't really call it a bad habit (this is the only way to do things in Java), but it's not "Pythonic". Python uses ``_`` it indicate a variable, method, function, etc is private and might change at any time, but doesn't enforce it. If the interface you'd like to present to the public differs from the one you'd like to manage internally, it's a perfect place for the setter/getter pattern with properties. Otherwise, just exposing the variable itself is fine.